int32 x
int32 y
int32 x_other
int32 y_other
float64 my_yaw
float64 dist
float64 yaw
int32 id
